src/GraphQL/Introspection/ISchemaComparer.cs

```diff
@@ -76,17 +76,17 @@ public class AlphabeticalSchemaComparer : ISchemaComparer
 
         private sealed class TypeByNameComparer : IComparer<IGraphType>
         {
-            public int Compare(IGraphType x, IGraphType y) => x.Name.CompareTo(y.Name);
+            public int Compare(IGraphType? x, IGraphType? y) => (x?.Name ?? "").CompareTo(y?.Name ?? "");
         }
 
         private sealed class DirectiveByNameComparer : IComparer<DirectiveGraphType>
         {
-            public int Compare(DirectiveGraphType x, DirectiveGraphType y) => x.Name.CompareTo(y.Name);
+            public int Compare(DirectiveGraphType? x, DirectiveGraphType? y) => (x?.Name ?? "").CompareTo(y?.Name ?? "");
         }
 
         private sealed class ArgumentByNameComparer : IComparer<QueryArgument>
         {
-            public int Compare(QueryArgument x, QueryArgument y) => x.Name.CompareTo(y.Name);
+            public int Compare(QueryArgument? x, QueryArgument? y) => (x?.Name ?? "").CompareTo(y?.Name ?? "");
```

Comment:
For some reason, there are annotations for some converters and there is no for others. Perhaps this is done specifically in GitHub UI in order not to throw huge lists of the annotations.

Reply:
Seems so.